content/altinn-studio/reference/data/external-api/_index.en.md

```
@@ -0,0 +1,436 @@
+
+Where:
+
+- `{org}` is the organization code (e.g., "ttd")
+- `{app}` is the application name (e.g., "my-app")
+- `{instanceOwnerPartyId:int}` is the party ID of the instance owner
+- `{instanceGuid:guid}` is the unique identifier of the instance
+- `{externalApiId}` is the ID of the External API client (the value returned by the `Id` property)
+
+For example, to access the "externalApiTestId" External API for a specific instance, you would use:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)